app/assets/stylesheets/application.css.scss

```diff
@@ -98,6 +98,24 @@ input {
   margin-top: 1%;
 }
 
+.field > input[type="email"], input[type="password"] {
```

Comment:
Avoid qualifying attribute selectors with an element.<br>Each selector in a comma sequence should be on its own line